2024-02-18 23:34:07,496 INFO    MainThread:1756 [wandb_setup.py:_flush():76] Current SDK version is 0.16.2
2024-02-18 23:34:07,496 INFO    MainThread:1756 [wandb_setup.py:_flush():76] Configure stats pid to 1756
2024-02-18 23:34:07,496 INFO    MainThread:1756 [wandb_setup.py:_flush():76] Loading settings from C:\Users\bdm2k\.config\wandb\settings
2024-02-18 23:34:07,497 INFO    MainThread:1756 [wandb_setup.py:_flush():76] Loading settings from d:\2023-2024\Semester 2\Data mining\Decision tree\wandb\settings
2024-02-18 23:34:07,497 INFO    MainThread:1756 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-02-18 23:34:07,497 INFO    MainThread:1756 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-02-18 23:34:07,498 INFO    MainThread:1756 [wandb_init.py:_log_setup():526] Logging user logs to d:\2023-2024\Semester 2\Data mining\Decision tree\wandb\run-20240218_233407-fi8zqrcz\logs\debug.log
2024-02-18 23:34:07,498 INFO    MainThread:1756 [wandb_init.py:_log_setup():527] Logging internal logs to d:\2023-2024\Semester 2\Data mining\Decision tree\wandb\run-20240218_233407-fi8zqrcz\logs\debug-internal.log
2024-02-18 23:34:07,499 INFO    MainThread:1756 [wandb_init.py:init():566] calling init triggers
2024-02-18 23:34:07,499 INFO    MainThread:1756 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {'min_sample': 2, 'max_depth': 16}
2024-02-18 23:34:07,499 INFO    MainThread:1756 [wandb_init.py:init():591] re-initializing run, found existing run on stack: 410542l7
2024-02-18 23:34:07,501 INFO    MainThread:1756 [wandb_run.py:_finish():1962] finishing run mc0c0z/DecisionTree/410542l7
2024-02-18 23:34:07,501 INFO    MainThread:1756 [jupyter.py:save_history():445] not saving jupyter history
2024-02-18 23:34:07,501 INFO    MainThread:1756 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-02-18 23:34:07,501 INFO    MainThread:1756 [wandb_init.py:_jupyter_teardown():454] cleaning up jupyter logic
2024-02-18 23:34:07,501 INFO    MainThread:1756 [wandb_run.py:_atexit_cleanup():2202] got exitcode: 0
2024-02-18 23:34:07,502 INFO    MainThread:1756 [wandb_run.py:_restore():2185] restore
2024-02-18 23:34:07,502 INFO    MainThread:1756 [wandb_run.py:_restore():2191] restore done
2024-02-18 23:34:11,551 INFO    MainThread:1756 [wandb_run.py:_footer_history_summary_info():3857] rendering history
2024-02-18 23:34:11,552 INFO    MainThread:1756 [wandb_run.py:_footer_history_summary_info():3889] rendering summary
2024-02-18 23:34:11,559 INFO    MainThread:1756 [wandb_run.py:_footer_sync_info():3816] logging synced files
2024-02-18 23:34:11,647 INFO    MainThread:1756 [wandb_init.py:init():616] starting backend
2024-02-18 23:34:11,647 INFO    MainThread:1756 [wandb_init.py:init():620] setting up manager
2024-02-18 23:34:11,649 INFO    MainThread:1756 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn, using: spawn
2024-02-18 23:34:11,649 INFO    MainThread:1756 [wandb_init.py:init():628] backend started and connected
2024-02-18 23:34:11,656 INFO    MainThread:1756 [wandb_run.py:_label_probe_notebook():1294] probe notebook
2024-02-18 23:34:11,656 INFO    MainThread:1756 [wandb_run.py:_label_probe_notebook():1304] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-02-18 23:34:11,656 INFO    MainThread:1756 [wandb_init.py:init():720] updated telemetry
2024-02-18 23:34:11,658 INFO    MainThread:1756 [wandb_init.py:init():753] communicating run to backend with 90.0 second timeout
2024-02-18 23:34:13,300 INFO    MainThread:1756 [wandb_run.py:_on_init():2254] communicating current version
2024-02-18 23:34:13,487 INFO    MainThread:1756 [wandb_run.py:_on_init():2263] got version response upgrade_message: "wandb version 0.16.3 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2024-02-18 23:34:13,487 INFO    MainThread:1756 [wandb_init.py:init():804] starting run threads in backend
2024-02-18 23:34:13,615 INFO    MainThread:1756 [wandb_run.py:_console_start():2233] atexit reg
2024-02-18 23:34:13,615 INFO    MainThread:1756 [wandb_run.py:_redirect():2088] redirect: wrap_raw
2024-02-18 23:34:13,616 INFO    MainThread:1756 [wandb_run.py:_redirect():2153] Wrapping output streams.
2024-02-18 23:34:13,616 INFO    MainThread:1756 [wandb_run.py:_redirect():2178] Redirects installed.
2024-02-18 23:34:13,616 INFO    MainThread:1756 [wandb_init.py:init():847] run started, returning control to user process
2024-02-18 23:34:41,181 INFO    MainThread:1756 [wandb_setup.py:_flush():76] Current SDK version is 0.16.2
2024-02-18 23:34:41,181 INFO    MainThread:1756 [wandb_setup.py:_flush():76] Configure stats pid to 1756
2024-02-18 23:34:41,181 INFO    MainThread:1756 [wandb_setup.py:_flush():76] Loading settings from C:\Users\bdm2k\.config\wandb\settings
2024-02-18 23:34:41,182 INFO    MainThread:1756 [wandb_setup.py:_flush():76] Loading settings from d:\2023-2024\Semester 2\Data mining\Decision tree\wandb\settings
2024-02-18 23:34:41,182 INFO    MainThread:1756 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-02-18 23:34:41,182 INFO    MainThread:1756 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-02-18 23:34:41,183 INFO    MainThread:1756 [wandb_init.py:_log_setup():526] Logging user logs to d:\2023-2024\Semester 2\Data mining\Decision tree\wandb\run-20240218_233441-03dpoycq\logs\debug.log
2024-02-18 23:34:41,183 INFO    MainThread:1756 [wandb_init.py:_log_setup():527] Logging internal logs to d:\2023-2024\Semester 2\Data mining\Decision tree\wandb\run-20240218_233441-03dpoycq\logs\debug-internal.log
2024-02-18 23:34:41,183 INFO    MainThread:1756 [wandb_init.py:_jupyter_setup():472] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x0000023C7C9FFE50>
2024-02-18 23:34:41,183 INFO    MainThread:1756 [wandb_init.py:init():566] calling init triggers
2024-02-18 23:34:41,183 INFO    MainThread:1756 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {'min_sample': 2, 'max_depth': 21}
2024-02-18 23:34:41,183 INFO    MainThread:1756 [wandb_init.py:init():591] re-initializing run, found existing run on stack: fi8zqrcz
2024-02-18 23:34:41,185 INFO    MainThread:1756 [wandb_run.py:_finish():1962] finishing run mc0c0z/DecisionTree/fi8zqrcz
2024-02-18 23:34:41,185 INFO    MainThread:1756 [wandb_run.py:_atexit_cleanup():2202] got exitcode: 0
2024-02-18 23:34:41,185 INFO    MainThread:1756 [wandb_run.py:_restore():2185] restore
2024-02-18 23:34:41,186 INFO    MainThread:1756 [wandb_run.py:_restore():2191] restore done
2024-02-18 23:34:45,119 INFO    MainThread:1756 [wandb_run.py:_footer_history_summary_info():3857] rendering history
2024-02-18 23:34:45,119 INFO    MainThread:1756 [wandb_run.py:_footer_history_summary_info():3889] rendering summary
2024-02-18 23:34:45,126 INFO    MainThread:1756 [wandb_run.py:_footer_sync_info():3816] logging synced files
